### PR TITLE
Tweak localization page

### PR DESCRIPTION
**Pull request by tengqm**
Wrap long lines for ease of change tracking.

### CODE REVIEW

File: content/en/docs/contribute/localization.md [hunk 1]
```diff
@@ -57,8 +66,9 @@ equivalent fix by updating the localization you're working on.
 Please limit pull requests to a single localization, since pull requests that change
 content in multiple localizations could be difficult to review.
 
-Follow [Suggesting Content Improvements](/docs/contribute/suggest-improvements/) to propose changes to
-that localization. The process is very similar to proposing changes to the upstream (English) content.
+Follow [Suggesting Content Improvements](/docs/contribute/suggesting-improvements/)
+to propose changes to that localization. The process is very similar to proposing changes to the
+upstream (English) content.
```
Comment (Sea-n) [inline content/en/docs/contribute/localization.md:71]: It's tricky but the old one https://kubernetes.io/docs/contribute/suggest-improvements/ is correct.
The new link https://kubernetes.io/docs/contribute/suggesting-improvements/ cannot be found.

**Comment by Sea-n** [comment]
Not sure why this file have different filename and slug.

It's PR #19576 two years ago made by @celestehorgan, maybe they can give us a sight.

https://github.com/kubernetes/website/blob/3332857e517789aee6f61e8338872dade969457e/content/en/docs/contribute/suggesting-improvements.md?plain=1#L1-L4

File: content/en/docs/contribute/localization.md (hunk 1 above)
Comment (tengqm) [reply content/en/docs/contribute/localization.md:71]: Okay. I'm gonna remove the 'slug' then.

**Comment by Sea-n** [comment]
Thanks, I think it should be removed too. 😊

There are many internal (and probably external) documents linked to that article, I think it would be better to add a redirection as well.

**Pull request merged**